src/Parsing/Parser.hs

```diff
@@ -1,31 +1,22 @@
 {-# LANGUAGE ConstraintKinds, DataKinds, GADTs, TypeFamilies #-}
 module Parsing.Parser
 ( Parser(..)
--- * À la carte parsers
+  -- * Parsers
+  -- $abstract
+, SomeParser(..)
 , goParser
+, javaParser
+, javascriptParser
+, jsonParser
+, jsxParser
 , markdownParser
+, phpParser
+, pythonParserALaCarte
+, pythonParserPrecise
 , pythonParser
 , rubyParser
 , tsxParser
 , typescriptParser
-, phpParser
-  -- * Abstract parsers
-
-  -- $abstract
-, SomeParser(..)
-, goParser'
-, javaParser'
-, javascriptParser'
-, jsonParserPrecise'
-, jsxParser'
-, markdownParser'
-, phpParser'
-, pythonParserALaCarte'
-, pythonParserPrecise'
-, pythonParser'
-, rubyParser'
-, tsxParser'
-, typescriptParser'
```

Comment:
Despite what the diff looks like, these weren’t deleted; the concrete ones were, and we renamed these to take their place.